gdx/src/com/badlogic/gdx/maps/tiled/renderers/BatchTiledMapRenderer.java

```diff
@@ -92,24 +98,26 @@ public Rectangle getViewBounds () {
 	}
 
 	public BatchTiledMapRenderer (TiledMap map) {
-		this(map, 1.0f);
+		this(map, 1.0f, 1.0f);
 	}
 
-	public BatchTiledMapRenderer (TiledMap map, float unitScale) {
+	public BatchTiledMapRenderer (TiledMap map, float unitScaleX, float unitScaleY) {
```

Comment:
Would prefer to make this a new constructor, leaving the original to accommodate the far more common use case of uniform unit scale (and to avoid breaking existing code, if possible).